### PR TITLE
Stripe

### PR DESCRIPTION
**Pull request by liambarstad**
#### Description of changes:
Just a fix of one test

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,57 +7,55 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 require 'csv'
 
-#Item.destroy_all
```
Comment (dphilla) [inline db/seeds.rb]: @liambarstad why uncomment this here?

**Review by dphilla — DISMISSED** [review]
@liambarstad I think we need to leave the seed commented so we don't get that file up to heroku.

**Review by Aram-Anderson — COMMENTED** [review]
Looks good. I'm going to put up one more PR to change to the root to '/items' instead of login.

**Review by Aram-Anderson — APPROVED** [review]
Looks good

**Pull request merged**